style.css

```diff
@@ -621,3 +631,504 @@
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  #nav-button {
+    margin-right: 9.86%;
+    background-color: #3e3c3c;
+    border-style: none;
+    height: 55.55%;
+    width: 2.77%;
+    margin-left: 0.83%;
+  }
+
+  .section {
+    height: 91.725%;
+  }
+
+  #container {
+    background-image: url(./desktopimg/backgrounds/background1.png);
+    background-color: #1c1a19;
+  }
+
+  #box {
+    position: absolute;
+    padding-top: 0%;
+    width: 566px;
+    height: 739px;
+    padding-left: 9.93%;
+    padding-right: 50.76%;
+    justify-content: center;
+    padding-bottom: 7.39%;
+    gap: 3.007%;
+  }
+
+  #heading1 {
+    height: 21.65%;
+    width: 80%;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  #heading2 {
+    height: 3.27%;
+    width: 100%;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  #description {
+    height: 16.644%;
+    width: 100%;
+    font-size: 19.6px;
+    line-height: 32px;
+  }
+
+  #social {
+    flex-direction: column;
+    position: absolute;
+    left: 2.5%;
+    top: 30.34%;
+  }
+
+  .title {
+    display: flex;
+    width: 78.68%;
+    padding-top: 135px;
+    margin-left: 10.659%;
+    margin-right: 10.659%;
+    margin-bottom: 114px;
+    align-items: center;
+    justify-content: center;
+    gap: 2.11%;
+  }
+
+  .empty {
+    width: 70.09%;
+    border: 1px solid #212121;
+  }
+
+  #heading {
+    height: 100%;
+    width: 27.53%;
+    margin: 0;
+    font-size: 39px;
+    text-align: center;
+  }
+
+  .works {
+    min-height: 1810px;
+  }
+
+  .work-grid {
+    width: 80.27%;
+    margin-left: 9.86%;
+    margin-right: 9.86%;
+    margin-bottom: 7.202%;
+    height: 76.187vh;
+    min-height: 1379px;
+  }
+
+  .desk-work {
+    display: flex;
+    width: 100%;
+    height: 32.27%;
+  }
+
+  #yoga-img {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+  }
+
+  #yoga {
+    width: 59.169%;
+  }
+
+  .desk-yoga {
+    width: 40.831%;
+    height: 77.678%;
+    margin-left: 2.076%;
+  }
+
+  #heading3 {
+    margin: 0;
+    font-size: 40px;
+    line-height: 52px;
+    height: 13.793%;
+  }
+
+  .framepara {
+    height: 40.229%;
+    margin-top: 5.193%;
+    margin-left: 0;
+    margin-right: 0;
+    font-size: 21.2px;
+    line-height: 28px;
+    padding-bottom: 5.193%;
+  }
+
+  #tags {
+    height: 32.183%;
+    width: 66.294%;
+  }
+
+  #tagsul {
+    margin: 0;
+    width: 100%;
+    height: 35.714%;
+  }
+
+  .tags1 {
+    width: 100%;
+  }
+
+  #project-button {
+    margin: 7.834% 0 0 0;
+  }
+
+  .frame {
+    height: 65.725%;
+    padding-top: 2.077%;
+  }
+
+  .cards {
+    padding: 0;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    row-gap: 4.07%;
+    column-gap: 2.07%;
+    height: 100%;
+    justify-content: normal;
+    width: 100%;
+    margin: 0;
+  }
+
+  .card {
```

Comment:
Great job, you implemented the design very well. But there is a nuance you need to consider here. On the Figma template you will see a hand cursor. It means when you hover over the card, it should look like that. Kindly add hover state looks like your first card. Pleae remember to add See Project button to take the user to the project page. 👨‍🎨

<img width="788" alt="Screenshot 2023-02-24 at 8 57 32 PM" src="https://user-images.githubusercontent.com/25887253/221259613-2fad3d2c-99ec-4c4b-98ff-e6c0b82038b9.png">
<img width="1438" alt="Screenshot 2023-02-24 at 9 00 23 PM" src="https://user-images.githubusercontent.com/25887253/221259637-803ccc34-ff55-42b3-9ad8-b5d99f66b09c.png">